src/servers/jolt_project_settings.cpp

```diff
@@ -170,6 +171,7 @@ void JoltProjectSettings::register_settings() {
 	register_setting_ranged(PAIR_CACHE_DISTANCE, 0.001f, U"0,0.01,0.00001,or_greater,suffix:m");
 	register_setting_ranged(PAIR_CACHE_ANGLE, Math::deg_to_rad(2.0f), U"0,180,0.01,radians");
 
+	register_setting_ranged(WORLD_BOUNDARY_SIZE, 2000.0f, U"1,2000,or_greater,suffix:m");
```

Comment:
@jrouwe Seeing as how we now have a dedicated broad phase layer for these big objects anyway, is there any major downside to just cranking this up to something silly, like what Godot/Bullet uses? That way we could presumably just omit this setting altogether.

Obviously you'll still run into "natural" precision problems once you move far enough away from (0, 0, 0), but would a half-extent of (for example) `1e+8` force any additional problems, even if you're sticking reasonably close to (0, 0, 0)?

Reply:
The GetSupportingFace, GetTrianglesStart and indeed the Draw functions become quite inaccurate if you bump the value up that high. You're not calling the first 2 functions right now, but if you ever decide to implement CharacterVirtual then it will call into the GetSupportingFace function. I would be inclined to keep the setting for now.